src/index.js

```diff
@@ -688,7 +838,7 @@ const handler = (event, context) => {
     handleCloudWatchLogs(event, context, lintEnv, tagRegexMap);
   }
 
-  if (event.Records) {
+  if (event.Records || event.eventSource) {
```

Comment:
not needed right.

If you see both MSK and code commit in this file test/__snapshots__/codecommit_collector.test.js.snap
both have Records as a JSON KEY.